packages/core/useDraggable/index.ts

```diff
@@ -159,6 +199,106 @@ export function useDraggable(
       e.stopPropagation()
   }
 
+  const scrollConfig = toValue(autoScroll)
+  const defaultScrollConfig = { speed: 2, margin: 30, direction: 'both' }
```

Comment:
you could move this outside the function to avoid creating it every time (since it is constant)